### ISSUE TITLE
Module not found: Can't resolve 'react-chartjs-2' in 'C:\Users\User\react-app\covid-19\src'

### ISSUE THREAD

**Issue by ChalanaJanith** [CleverProgrammers/react-covid-tracker#15]
Hello sir,

 I was creating this project, I came across this error. 

./src/LineGraph.js
Module not found: Can't resolve 'react-chartjs-2' in 'C:\Users\User\react-app\covid-19-tracker\src'
Error from chokidar (C:\): Error: EBUSY: resource busy or locked, lstat 'C:\hiberfil.sys'

Sir, can you help me how to solve this? because this is my first React JS project.
![Screenshot 2020-11-06 071158](https://user-images.githubusercontent.com/56481810/98381955-65cd3180-1fff-11eb-98bf-59de901afe3f.jpg)

**Comment by Akr1s** [comment]
I think you need to install package `react-chartjs-2` using npm

**Comment by ChalanaJanith** [comment]
> I think you need to install package `react-chartjs-2` using npm

Yeah it should be installed inside the SRC folder... Anyway Thanks Brother For your Reply,
